introduction.md

```diff
@@ -0,0 +1 @@
+My name is Wonkun Kim who is a software engineer working at Schneider Electric in Raleigh, NC.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.